README.md

```diff
@@ -1 +1,74 @@
-# Capstone-project
\ No newline at end of file
+
+# Project Name
+ Capstone-project
+
+## Description
+
+This is my Capstone project that shows how much I learned trough the first Module. Look through some of my work and experience! If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it.
+
+## 🖥️ Live Demo
+
+[My Portfolio](https://elhamatokhi.github.io/Capstone-project/)
+
+## Built With
+HTML
+CSS(scss)
+JavaScript
+### Setup
+1. Download the **Zip** file or clone the repo with:
+```bash
+git clone https://github.com/elhamatokhi/Capstone-project.git
+```
+3. To access cloned directory run:
+```bash
+cd MyPortfolio
+```
+3. Open it with the live server
+
+### Install
+> To install linters and other project's dependencies run:
+```bash
+npm install
+```
+## Run Test for Linters
+
+For tracking linter errors locally you need to follow these steps:
+
+1. For tracking linter errors in HTML files run:
+```bash 
+npx hint .
+```
+
+2. For tracking linter errors in CSS or SASS files run:
+
+```bash
+npx stylelint "\*_/_.{css,scss}"
+```
+
+3. For tracking linter errors in JavaScript files run:
+
+```bash
+npx eslint .
+```
+
+## Authors
+👤 **Author1**
+
+- GitHub:[@elhamatokhi](https://github.com/elhamatokhi)
+- 
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```

Comment:
- [ ] You should give credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).